backend/app/adapter/gqlapi/resolver/authmutation.go

```diff
@@ -67,9 +67,9 @@ func (a AuthMutation) CreateURL(args *CreateURLArgs) (*URL, error) {
 	case url.ErrAliasExist:
 		return nil, ErrURLAliasExist(*customAlias)
 	case url.ErrInvalidLongLink:
-		return nil, ErrInvalidLongLink(u.LongLink)
+		return nil, ErrInvalidLongLink{u.LongLink, err.(url.ErrInvalidLongLink).Violation.String()}
 	case url.ErrInvalidCustomAlias:
-		return nil, ErrInvalidCustomAlias(*customAlias)
+		return nil, ErrInvalidCustomAlias{*customAlias, err.(url.ErrInvalidCustomAlias).Violation.String()}
```

Comment:
It was awhile back when I created Short. Now, in Go 13, `Is` and `As` are added to help check error. Please add a TODO to help migrate to the new syntax: https://blog.golang.org/go1.13-errors

Reply:
TODO
- [ ] Add TODO to migrate switch statement for error type checking to user `errors.As` instead

Reply:
My bad, I forgot to address this. TODO has been added and my next commit will contain the TODO comments.